lib/Models/Catalog/Ows/SensorObservationServiceCatalogItem.ts

```diff
@@ -104,7 +104,8 @@ class SosAutomaticStylesStratum extends TableAutomaticStylesStratum {
         }),
         pointSize: createStratumInstance(TablePointSizeStyleTraits, {
           pointSizeColumn: p.identifier
-        })
+        }),
+        hidden: false
```

Comment:
Why do we need this here? Could you add a comment?

Reply:
SOS was implemented long before we got selectable dimensions, so the frequency selector is implemented a "style" when it should actually just be a plain selectable dimension. Style selector is [hidden by default](https://github.com/TerriaJS/terriajs/blob/main/lib/Table/TableStyle.ts#L82) when the table uses less than 1 color. We set `hidden: true` to override that behavior and show the selector.

Ideal fix is to rewrite the selectors using simple selectable dimensions. But I think in the interest of time that should be a new PR.